packages/truss/src/methods.ts

```diff
@@ -38,17 +40,19 @@ export function newParamMethod(abbr: UtilityName, prop: keyof Properties): Utili
  * @param defs a map of abbreviation name --> value
  * @param baseName the base name to use, i.e. `mt`
  * @param includePx generate an extra `${baseName}Px` method that calls the base method with a converted px value
+ * @param baseDefs additional properties to set for the base method
  */
 export function newMethodsForProp<P extends Prop>(
   prop: P,
-  defs: Record<UtilityName, Properties[P]>,
+  defs: Record<UtilityName, Properties[P] | Properties>,
```

Comment:
@allangaldinosilva why is the `| Properties` necessary? Is this to support the extra props that aren't technically for `Properties["lineClamp"]`?

If so, we should probably just remove `Propeties[P]` and put in an in-source comment that we're accepting the full `Properties` (not restricted to `Properties[P]` so that we can accept extra properties.

Reply:
Oh, wait, I see, sometimes `Properties[P]` is a specific value, and sometimes it's an open-ended up object literal...

^ might make a good in-source comment :-)

Reply:
in-source comment added :)